src/core/src/main/scala/play/api/db/slick/SlickApi.scala

```diff
@@ -48,7 +49,10 @@ final class DefaultSlickApi @Inject() (
         playConfig.get[Map[String, Config]](slickDbKey)
       } else Map.empty[String, Config]
     }
-    (for ((name, config) <- configs) yield (DbName(name), new DatabaseConfigFactory(name, config, lifecycle)))(collection.breakOut)
+    {
+      for ((name, config) <- configs)
+        yield (DbName(name), new DatabaseConfigFactory(name, config, lifecycle))
+    }.to(Map).asInstanceOf[Map[DbName, DatabaseConfigFactory]]
```

Comment:
This is odd, but this is simply not working for Scala 2.11 and 2.12

Basically, when doing the following...

```scala
import scala.collection.compat._
val ys = Seq(1 -> "a", 2 -> "b")
val m: Map[Int, String] = m.to(Map)
```

It compiles in Scala 2.13, but fails for 2.11 and 2.12. To have it working in those version we need a explicitly cast.

See: https://github.com/scala/scala-collection-compat/blob/master/compat/src/test/scala/test/scala/collection/CollectionTest.scala#L37-L42